march_shared_classes/src/march_shared_classes/gait/subgait.py

```diff
@@ -298,6 +298,16 @@ def get_joint_names(self):
         """Get the names of all the joints existing in the joint list."""
         return [joint.name for joint in self.joints]
 
+    @property
+    def starting_position(self):
+        """Returns a dictionary of joint positions at the start of this subgait."""
+        return dict([(joint.name, joint.setpoints[0].position) for joint in self.joints])
```

Comment:
```suggestion
        return {joint.name: joint.setpoints[0].position for joint in self.joints}
```
It think this is the prettier way to do it.